app/server/events/normalizer.py

```
@@ -143,3 +143,269 @@ def normalize(
+        issue = raw_payload.get("issue", {})
+        key = issue.get("key") or raw_payload.get("key") or "JIRA-000"
+        fields = issue.get("fields", {})
+        summary = fields.get("summary", "")
+        
+        status_data = fields.get("status", {})
+        status_name = status_data.get("name", "").lower()
+        
+        priority_data = fields.get("priority", {})
+        priority_name = priority_data.get("name", "").lower()
+
+        labels = [str(l).lower() for l in fields.get("labels", [])]
```

Ruff (0.16.3)

[error] Ambiguous variable name: `l`

(E741)